.travis.yml

```diff
@@ -36,7 +47,11 @@ before_install:
   # Does it already exist? Who owns? What permissions?
   - ls -lah ~/.npm || mkdir ~/.npm
   # Make sure we own it
-  - sudo chown -R $USER ~/.npm
+  - chown -Rc $USER ~/.npm
+  # Build boost ourselves. Maintenance note: to move to a new version
+  # of boost, update BOOST_ROOT in the env: section, and the
+  # URL and paths used in the bin/sh/install-boost.sh script.
+  - bash bin/sh/install-boost.sh
```

Comment:
Shouldn't we be trying to build static libraries by default?  that's our policy in SConstruct.